src/shared/parsers/CellParser.parser.tsx

```diff
@@ -0,0 +1,35 @@
+// TODO: fix cell any type.
+export const renderCell = (cell, key: string) => {
+  if (key === 'image') {
```

Comment:
look like should extact const
```suggestion

  if (key === CELL_KEY.IMAGE) {
```

Reply:
fixed